### PR TITLE
RAVTDBQ_281_Support_For_Otranslate_Function

### PR DESCRIPTION
**Pull request by Pritam-Gote**
Fix_For_Otranslate_Function

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java [hunk 1]
```diff
@@ -1457,4 +1457,14 @@ private SqlLibraryOperators() {
           OperandTypes.family(SqlTypeFamily.ANY,
               SqlTypeFamily.STRING),
           SqlFunctionCategory.TIMEDATE);
+
+  @LibraryOperator(libraries = {BIG_QUERY, HIVE, SPARK, SNOWFLAKE})
```
Comment (Mahesh-Raut) [inline core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java:1461]: Please add a test for the TRANSLATE function in calcite.

File: core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java [hunk 2]
```diff
@@ -1457,4 +1457,14 @@ private SqlLibraryOperators() {
...
+  public static final SqlFunction TRANSLATE =
```
Comment (Mahesh-Raut) [inline core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java:1462]: Please add a test for the TRANSLATE function in calcite.

Comment (Pritam-Gote) [reply core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java:1462]: Done

**Comment by Mahesh-Raut** [comment]
L1 is done. Looks good.

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 3]
```diff
@@ -11706,4 +11706,29 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
...
+        builder.literal("scott"), builder.literal("t"), builder.literal("a"));
+    RelNode root = builder
+        .project(rexNode)
+        .build();
+    final String expectedBQSql = "SELECT TRANSLATE('scott', 't', 'a') AS `$f0`"
+        + "\nFROM scott.EMP";
+    assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBQSql));
+  }
+
+  @Test public void testTranslateWithNumberParameter() {
```
Comment (Shubham-Isalkar) [inline core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:11722]: As discussed, Please add 2 scenarios where the 2nd parameter length < 3rd parameter length in OTRANSLATE function and vice versa.

**Review by Shubham-Isalkar — APPROVED** [review]
L2 Done. The changes look good.
Proceed if data testing is clear.

**Pull request merged**